Lib/json/tool.py

```diff
@@ -24,20 +24,30 @@ def main():
                         help='a JSON file to be validated or pretty-printed')
     parser.add_argument('outfile', nargs='?', type=argparse.FileType('w'),
                         help='write the output of infile to outfile')
+    parser.add_argument('--indent', default=4,
+                        help='the indentation for pretty-print (int or str)')
+    parser.add_argument('--no-ensure-ascii', dest='ensure_ascii', action='store_false',
+                        help='disable escaping of non-ASCII characters')
     parser.add_argument('--sort-keys', action='store_true', default=False,
                         help='sort the output of dictionaries alphabetically by key')
     options = parser.parse_args()
 
     infile = options.infile or sys.stdin
     outfile = options.outfile or sys.stdout
     sort_keys = options.sort_keys
+    try:
+        indent = int(options.indent)
+    except ValueError:
```

Comment:
if you use type in argparse you can remove this section

Reply:
No, then it will be impossible to pass tab character to json dumps. This is a feature - for example `--indent="	"`.

Reply:
ack!